util/gen-typescript-decl.ts

```diff
@@ -0,0 +1,254 @@
+/**
+ * Generate TypeScript decl for options in node-libtidy
+ *
+ * @author Wang Guan <momocraft@gmail.com>
+ *
+ * vim: shiftwidth=2
+ */
+
+import * as fs from 'fs';
+import * as path from 'path';
+import * as dom from 'dts-dom';
+import { expect } from 'chai';
+
+// a bootstrap declaration is required to call libtidy.TidyDoc.getOptionList()
+import * as libtidy from '../';
+
+/**
+ * Main entrypoint: generate and update ../src/index.d.ts
+ */
+function main() {
+  const dts = path.join(__dirname, ...'../src/index.d.ts'.split('/'));
+  const oldContent = fs.readFileSync(dts, "utf-8");
+  const header = oldContent.split('// START-GENERATED-NAMESPACE')[0];
+  var newContent = header + `// START-GENERATED-NAMESPACE
+// The rest of this file is generated.
+` + generate();
+
+  if (newContent === oldContent) {
```

Comment:
I'm getting a little confused here: if we are not including generated code in the repo, is a "verify mode" still needed?